rasa_nlu/extractors/crf_entity_extractor.py

```diff
@@ -120,9 +146,19 @@ def _create_dataset(self, examples):
             dataset.append(self._from_json_to_crf(example, entity_offsets))
         return dataset
 
+    def _check_spacy_doc(self, message):
+        if self.pos_features and message.get("spacy_doc") is None:
+            raise InvalidConfigError(
+                'Could not find `spacy_doc` attribute '
```

Comment:
same, what do they need to fix it (e.g. most likely add `ner_spacy` to the pipeline)